catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -265,6 +272,8 @@ private void addMetacardDates(final XstreamPathValueTracker pathValueTracker,
         metacard.setModifiedDate(date);
         metacard.setCreatedDate(date);
         metacard.setEffectiveDate(date);
+        metacard.setAttribute(Core.METACARD_MODIFIED, date);
```

Comment:
This method is mapping the metadata dates and this attribute is correct in my opinion.  The Resource dates can be mapped to `//MD_Metadata/identificationInfo/MD_DataIdentification/citation/CI_Citation/date`.  We should consult the GMD spec to determine if these can be gco:Date *AND* gco:DateTime.  If this is true, then these fields will have to be parsed differently.  See https://github.com/codice/alliance/pull/110